courses/fundamentals_of_ada/060_record_types.rst

```diff
@@ -3,6 +3,17 @@
 Record Types
 **************
 
+.. |rightarrow| replace:: :math:`\rightarrow`
+
+.. role:: ada(code)
+   :language: ada
+
+.. role:: C(code)
+   :language: C
+
+.. role:: cpp(code)
+   :language: C++
+
```

Comment:
Would it be better to create a common "include" file that defines things like roles and substitutions?